packages/taquito-http-utils/README.md

```diff
@@ -1,21 +1,48 @@
+`method`(string): HTTP method of the request
+`timeout`(number): request timeout 
+`json`(boolean): Parse response into JSON when set to `true`; defaults to `true`
+`query`(object): Query that we would like to pass as an HTTP request
+`headers`(object): HTTP request header 
+`mimeType`(string): Sets the MIME type of the request; see [MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types)
+
+
+## Install
+The package(s) need to be installed as follows
+```
+npm install @taquito/http-utils
+```
+
+## Usage
+Create an instance of `HttpBackend` and call it's member function `createRequest` to construct an HTTP request. 
 ```ts
 import { HttpBackend } from '@taquito/http-utils';
 
-const http = new HttpBackend();
-```
+const httpBackend = new HttpBackend();
+const response = httpBackend.createRequest<string>({
+    url: `/chains/${chain}/blocks/${block}/context/contracts/${address}/script`,
+    method: 'GET',
+    mimeType: "text; charset=utf-8",
+    json: false
+});
 
-See the top-level [https://github.com/ecadlabs/taquito](https://github.com/ecadlabs/taquito) file for details on reporting issues, contributing and versioning.
+```
 
-## API Documentation
+## Additional Info
+See the top-level https://github.com/ecadlabs/taquito file for details on reporting issues, contributing, and versioning.
 
-TypeDoc style documentation is available on-line [here](https://tezostaquito.io/typedoc/modules/_taquito_http_utils.html)
 
 ## Disclaimer
 
-THIS SOFTWARE IS PROVIDED "AS IS" AND ANY EXPRESSED OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE REGENTS OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
\ No newline at end of file
+THIS SOFTWARE IS PROVIDED "AS IS" AND ANY EXPRESSED OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE REGENTS OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.# Taquito HTTP Utilities package
```

Comment:
# Taquito HTTP Utilities package  seems out of place here

Reply:
good catch, that was mistakenly copy-pasted